### PR TITLE
Add intent for using Discussions feature in repo

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

Add guidance on intent for enabling Discussions feature

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

**Comment by SteveL-MSFT** [comment]
The static analysis failure is due to the discussions URL not be active yet

**Comment by daxian-dbw** [comment]
Can we have both the blog and this addition to README?
I think a blog may draw more attention. README can point to that blog.

**Comment by SteveL-MSFT** [comment]
We discussed not having the blog as the content is not the direction we want to go being too prescriptive and creating a new process for using discussions

File: README.md [hunk 1]
```diff
@@ -143,6 +143,18 @@ To install a specific version, visit [releases](https://github.com/PowerShell/Po
...
+
+[GitHub Discussions](https://docs.github.com/en/free-pro-team@latest/discussions/quickstart) is a feature to enable fluid and open discussions within the community
+for topics that are not related to code, unlike issues.
+
+This is an experiment we are trying in our repositories to see if it helps move discussions out of issues so that issues remain actionable by the team or members of the community.
+There should be no expectation that PowerShell team members are regular participants in the discussions.
+Individual PowerShell team members may choose to participate in discussions, but the expectation is that community members help drive discussions so that team members
+can focus on issues.
+
+Create or join a [discussion](https://github.com/PowerShell/PowerShell/discussions).
```
Comment (iSazonov) [inline README.md:156]: I think we need a reflection between discussion categories and issue areas.

**Comment by iSazonov** [comment]
Adding this feature makes sense only if MSFT team intends (1) to _actively participate_ in discussions and (2) make it the _main forum_ for PowerShell, otherwise it will be _just another forum_, which does not make sense since there are already many great sites with their audience.

**Comment by SteveL-MSFT** [comment]
@PowerShell/powershell-committee discussed this, we agree with the intent to enabling discussions to move those out of issues.  By the nature of Discussions being on GitHub, we believe it's more likely PowerShell team members will participate, but also we want to set no expectation that PowerShell team members are expected to participate.

**Review by joeyaiello — APPROVED** [review]
Language looks good for the intent

@iSazonov I represented your position for a while in our conversation within the Committee today, but I was ultimately persuaded by the argument @SteveL-MSFT made in the comment above. More specifically that:
* this is an experiment, and we should keep your question in mind as we decide whether to leave Discussions on
* the proximity of the Discussions tab to the code repo will likely *encourage* more contribution from PS Team members than other forums, in spite of the lack of *requirement*

Additionally, I don't think any of this should dissuade folks from using existing discussion forums like SO, PS.org, Discord/Slack UG, Reddit, etc.

**Comment by daxian-dbw** [comment]
The static analysis failed because the discussion tab hasn't been enabled. The URL will exist once that's done.

**Pull request merged**